www/include/Administration/parameters/DB-Func.php

```diff
@@ -113,6 +113,21 @@ function is_writable_file_if_exist($path = null)
     return false;
 }
 
+/**
+ * rule to check the session duration value chosen by the user
+ * @param int $value
+ * @param int $valueMax
+ *
+ * @return bool
+ */
+function is_session_duration_valid(int $value = null)
+{
+    return ($value > 0 && $value <= SESSION_DURATION_LIMIT);
+        return true;
+    }
+    return false;
```

Comment:
Please remove these lines following the above changes ;-)

Reply:
done